app/views/users/show.html.erb

```diff
@@ -1 +1,15 @@
 <h1>Here are the details of the selected user.</h1>
+
+<%= render @user %>
+
+<% @posts.each do |post| %>
+    <p>
+    <strong>Title:</strong>
+    <%= post.title %>
+  </p>
+
+  <p>
+    <strong>Text:</strong>
+    <%= post.text %>
+  </p>
+<% end %>
```

Comment:
- [ ] Please your user show file should match the given routes `http://127.0.0.1:3000/users/1` [wireframes](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=ABIFLA3SVPWT6MXQ3C2OGFLBHIHES)